Objects/moduleobject.c

```diff
@@ -177,6 +178,7 @@ new_module_notrack(PyTypeObject *mt)
     m->md_state = NULL;
     m->md_weaklist = NULL;
     m->md_name = NULL;
+    m->m_dict_version = 0;
```

Comment:
All other fields start with `md_`, can this use that as well?

Reply:
I think this got added when we reified the whole dict object on access, we should remove it.